index.bs

```diff
@@ -1524,17 +1532,36 @@ When this method is invoked, the [=authenticator=] must perform the following pr
     <figcaption>Generating an [=assertion signature=].</figcaption>
 </figure>
 
-On successful completion, the authenticator returns to the user agent:
+On successful completion, the [=authenticator=] returns to the user agent:
 - The identifier of the credential (credential ID) used to generate the [=assertion signature=].
 - The [=authenticator data=] used to generate the [=assertion signature=].
 - The [=assertion signature=].
 
-If the authenticator cannot find any credential corresponding to the specified [=[RP]=] that matches the specified criteria, it
+If the [=authenticator=] cannot find any credential corresponding to the specified [=[RP]=] that matches the specified criteria, it
 terminates the operation and returns an error.
 
 If the user refuses consent, the authenticator returns an appropriate error status to the client.
+    </dd>
+    <dt>"public-key-uaf"</dt>
+    <dd>
+- Follow the procedure as specified in [[FIDO-UAF-Authenticator-Commands]] section 6.3. Using
```

Comment:
Step 4 of this algorithm (above) is now "Prompt the user to select a credential from among the above list."
Step 7.1 of https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-uaf-authnr-cmds-v1.1-id-20170202.html#command-description-2 (which you're saying to execute here) is "If this authenticator has a user interface and wants to use it for this purpose: Ask the user which of the usernames he wants to use for this operation. Select the related RawKeyHandle and jump to step #8."

So you're specifying that a UAF authenticator needs to ask the user twice. That's obviously wrong.

Please imagine that you don't know anything about FIDO or UAF, but you want to implement a client and authenticator anyway. Write down the steps your implementation would need to follow in order to behave the same as all the other implementations folks are producing.